### DISCUSSION TITLE
TailwindCSS sticky example shows sticky header not or late

### DISCUSSION THREAD

**Discussion by git-no** [tailwindlabs/tailwindcss#10421] [General]
The TailwindCSS [Sticky Position Example Code](https://tailwindcss.com/docs/position#sticky-positioning-elements) or coded in an own webpage shows during and after scrolling instead the sticky title, e.g. B, C etc. blank or the title very late. All other table cells do scroll just in time.

Is this a TailwindCSS / CSS issue!?

Happens minimum with latest TailwindCSS version and latest Chrome Browser (tested in macOS) version.

**Comment by wongjn** [comment]
I can reproduce some strange behavior too! The headings appear and disappear at certain times, or only after interacting with them with the mouse. Removing the `backdrop-blur-sm` class seems to rectify the issues – seems like it could be a rendering bug in Chrome?
